Bindings/bindings.json

```diff
@@ -7,23 +7,67 @@
         "triggerRefHelp": "You can use brackets ({value}) and percent symbols (%value%) to format your path based on parameters from the trigger object."
     },
     "bindings": [
-        {
-            "type": "timerTrigger",
-            "displayName": "Timer",
-            "direction": "trigger",
+      {
```

Comment:
Strange, VS default was 4 for me. But in any case, let's not fix in same file :)